### DISCUSSION TITLE
[WASM] How to get rid of WebSocket errors and warnings?

### DISCUSSION THREAD

**Discussion by DierkDroth** [unoplatform/uno#8271] [Q&A] [answered]
How could I get rid of the websocket errors and warning from attached log file?
I also attached my launchSettings.json file.

Thanks in advance

[localhost-1646731337500.log](https://github.com/unoplatform/uno/files/8204566/localhost-1646731337500.log)
[launchSettings.zip](https://github.com/unoplatform/uno/files/8204578/launchSettings.zip)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report. For now, aside from removing the "Uno.UI.RemoteControl" package, there no way to remove those messages.

**Comment by DierkDroth** [comment]
Thanks @jeromelaban. 

What would I lose if I removed "Uno.UI.RemoteControl"? Could I still debug?

**Reply by jeromelaban** [reply]
You'd only lose the support for Hot Reload, debugging is another package.

**Reply by DierkDroth** [reply]
Great, thanks @jeromelaban - I'm not using Hot Reload anyway...

However, there still is one WebSocket error in the logs (attached). Any idea where this is coming from?
[localhost-1646754106628.zip](https://github.com/unoplatform/uno/files/8207558/localhost-1646754106628.zip)

**Reply by jeromelaban** [reply]
This one comes from visual studio, it's not something we can adjust.

**Reply by DierkDroth** [reply]
Understand. Thanks @jeromelaban
